src/components/Loader/people/PeopleRow.tsx

```diff
@@ -0,0 +1,49 @@
+import cn from 'classnames';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../../../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  person: Person,
+  slug: string | undefined,
+};
+
+export const PeopleRow: React.FC<Props> = ({ person, slug }) => {
+  return (
+    <tr
+      data-cy="person"
+      key={person.slug}
+      className={cn({
+        'has-background-warning': slug === person.slug,
+      })}
+    >
+      <td>
+        <Link
```

Comment:
As I understand it should be a separate component (check create the <PersonLink person={person} /> and use it for all existing people;)